javascript/house-style.md

```diff
@@ -871,3 +871,41 @@ The `utils` directory is used to house JavaScript written for the project that d
   - It can expose functions and/or prototypal classes
 
 An example utility might be a function to make a string title-case.
+
+### Be careful when transpiling
+
+As exciting it can be to use modern Javascript syntax, it remains our
+responsibility as web developers to make sure our benefits do not harm our user
+experience. Any negative impact on performance affects the user experience.  
+Transpiling does increase Javascript bundles size.  
```

Comment:
```suggestion
As enticing it can be to use modern Javascript syntax, it remains the developers responsibility to ensure the developer experience does not negatively impact the user experience, e.g. by decreasing performance. 

Transpiling can easily increase the JavaScript bundle size, by incorporating polyfills to for supported browsers.

```